source/solvers/postprocessing_cfd.cc

```diff
@@ -592,9 +594,17 @@ calculate_forces(
                                     }
                                   shear_rate = velocity_gradients[q] +
                                                transpose(velocity_gradients[q]);
-                                  viscosity = rheological_model->get_viscosity(
+
+                                  const double shear_rate_magnitude =
                                     rheological_model->get_shear_rate_magnitude(
-                                      shear_rate));
+                                      shear_rate);
+
+                                  std::map<field, double> field_values;
```

Comment:
Isn't it better to define field_values as a member variable?

Reply:
What do you mean as a member variable? This is not a class.